docs/advanced/input_files/input-main.md

```diff
@@ -3917,6 +3917,15 @@ These parameters are used to solve the excited states using. e.g. LR-TDDFT.
 Currently supported: `RPA`, `LDA`, `PBE`, `HSE`, `HF`.
 - **Default**: LDA
 
+### lr_init_xc_kernel
+
+- **Type**: String
+- **Description**: The method to initalize the xc kernel. 
+  - "from_chg_groundstate": Calculate xc kerenel ($f_\text{xc}$) from the ground-state charge density.
```

Comment:
@maki49 @mohanchen 
I carefully considered the keywords design for this parameter. I think there are better candidates, based on my experience from various dft implementation. My suggestions are:
1. use `default` or `from_charge` or `scf` instead of `from_chg_groundstate`. If I understand correctly, this option will get charge density in runtime from ABACUS ground-state calculation (mainly ESolver_KS_*), and switch to LR-TDDFT calculation workflow. Because the `chg_groundstate` actually reflects the technical detail that "the algorithm will use charge denstiy of ground state, then calculate fxc, ...", but this is too much for user to know, and not brief enough so that it is not directly the **method** of initializing the xc kernel. In its corresponding explanation, you can keep contents there, as "calculate ... from the ground-state charge denstiy".
2. use `file` as it is, but please consider if the second and third have some redundancy.
3. use `restart_from_charge` or `nscf` instead of `from_chg_file`. The reason for the former is based on the former two, no abbr. is used, so this option should keep accordance with them, the latter corresponds to the option 1's `scf`. It is rarely seen the ABACUS starts directly with charge, or it can be a kind of run from converged charge density-like behavior, then it is a kind of `restart` or `nscf`.

;)

Reply:
1. I will choose "default". It means the read-in charge density should be consistent with the read-in ground-state wavefunction to calculate all the integrals. 
2. Actually 1 (nspin=1) or 3 (nspin=2) files will be read in. I will change the description.
3. For no abbr. I think "from_charge_file" is alright. I feel "scf" and "nscf" is kind of misleading... "scf" sounds okay for option 1 as it means the charge to calculate $f_\text{xc}$ is from SCF calculation, but this option here has nothing to do with where the charge file come from or whether it is consistent with the wavefunction. "restart" is more confusing because LR-TDDFT does completely different calculation from DFT.